packages/grafana-ui/src/components/NodeGraph/ViewControls.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Button } from '../Button';
+import { stylesFactory, useTheme } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  minusButton: css`
+    margin-left: 2px;
+  `,
+
+  scale: css`
+    font-size: 10px;
+    color: ${theme.colors.textFaint};
+  `,
+
+  scrollHelp: css`
+    font-size: 8px;
+    color: ${theme.colors.textFaint};
+  `,
+}));
+
+interface Props<Config> {
+  config: Config;
+  onConfigChange: (config: Config) => void;
+  onPlus: () => void;
+  onMinus: () => void;
+  scale: number;
+}
+
+/**
+ * Control buttons for zoom but also some layout config inputs mainly for debugging.
+ */
+export function ViewControls<Config extends Record<string, any>>(props: Props<Config>) {
+  const { config, onConfigChange, onPlus, onMinus, scale } = props;
+  const [showConfig, setShowConfig] = useState(false);
+  const styles = getStyles(useTheme());
+
+  // For debugging the layout, should be removed here and maybe moved to panel config later on
+  const allowConfiguration = false;
+
+  return (
+    <>
+      <Button icon={'plus-circle'} onClick={onPlus} size={'sm'} title={'Zoom in'} />
+      <Button className={styles.minusButton} icon={'minus-circle'} onClick={onMinus} size={'sm'} title={'Zoom out'} />
```

Comment:
according to https://developers.grafana.com/ui/latest/index.html?path=/docs/buttons-button--simple#secondary:

> When there is no main important action on a given page, all `Button` components should use the secondary style.

```suggestion
      <Button icon="plus-circle" onClick={onPlus} size="sm" title="Zoom in" variant="secondary" />
      <Button
        className={styles.minusButton}
        icon="minus-circle"
        onClick={onMinus}
        size="sm"
        title="Zoom out"
        variant="secondary"
      />```

Reply:
also, maybe we could use `HorizontalGroup` which will give elements a predefined (and consistent) spacing without the need of the custom spacing for the minus button:
```ts
      <HorizontalGroup spacing="xs">
        <Button icon="plus-circle" onClick={onPlus} size="sm" title="Zoom in" variant="secondary" />
        <Button icon="minus-circle" onClick={onMinus} size="sm" title="Zoom out" variant="secondary" />
      </HorizontalGroup>
```

![Screenshot 2021-01-19 at 10 34 02](https://user-images.githubusercontent.com/1170767/105022935-3d6fa380-5a42-11eb-91e1-3dbaad02f482.png)